config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -75,11 +91,117 @@ processors:
         action: insert
       - key: aws.ecs.task.id
         action: delete
-      - key: TaskDefinitionFamily
+      - key: Family
         from_attribute: aws.ecs.task.family
         action: insert
       - key: aws.ecs.task.family
         action: delete
+      - key: TaskARN
+        from_attribute: aws.ecs.task.arn
+        action: insert
+      - key: aws.ecs.task.arn
+        action: delete
+      - key: DockerName
+        from_attribute: aws.ecs.docker.name
+        action: insert
+      - key: aws.ecs.docker.name
+        action: delete
+      - key: Revision
+        from_attribute: aws.ecs.task.version
+        action: insert
+      - key: aws.ecs.task.version
+        action: delete
+      - key: PullStartedAt
+        from_attribute: aws.ecs.task.pull_started_at
+        action: insert
+      - key: aws.ecs.task.pull_started_at
+        action: delete
+      - key: PullStoppedAt
+        from_attribute: aws.ecs.task.pull_stopped_at
+        action: insert
+      - key: aws.ecs.task.pull_stopped_at
+        action: delete
+      - key: AvailabilityZone
+        from_attribute: cloud.zone
+        action: insert
+      - key: cloud.zone
+        action: delete
+      - key: LaunchType
+        from_attribute: aws.ecs.task.launch_type
+        action: insert
+      - key: aws.ecs.task.launch_type
+        action: delete
+      - key: KnownStatus
```

Comment:
For the fields which are common in both Task and Container level metrics, we need to discuss with PM. We can do two things- lets remove them from this version. It would make the merge faster. Or, we can talk to the PM first, finalize the them and then merge.

Reply:
Thanks Rayhan for the email, but haven't received reply from PMs for now, so deleted these two fields.